Phase1ScanHistoMaker.cc

```diff
@@ -507,6 +517,7 @@ int main(int argc, char* argv[]) {
   sh.AddHistos("clust", { .fill="ROC_Ring2_BladePanel_vs_Disk",                        .pfs={"DelayScans"},          .cuts={"ZeroBias","Ring2"}, .draw="COLZ", .opt="Log", .ranges={0,0, 0,0, 0,0} });
 #endif
   
+  std::function<void()> add_special_histos = [&sh]{};
```

Comment:
@TizianoBevilacqua I had to move this since in the original the code broke if I switched to HV scan mode (#define HV_Scan     1). Please look at this special histos part if it's okay this way.

Reply:
By just looking at it it seems fine, have you tested it?